src/Mod/Part/App/ShapeFix/ShapeFix_EdgePy.xml

```diff
@@ -18,37 +18,120 @@
     </Documentation>
     <Methode Name="fixRemovePCurve">
       <Documentation>
-        <UserDocu></UserDocu>
+        <UserDocu> Removes the pcurve(s) of the edge if it does not match the
+   vertices
+   Check is done
+   Use    : It is to be called when pcurve of an edge can be wrong
+   (e.g., after import from IGES)
+   Returns: True, if does not match, removed (status DONE)
+   False, (status OK) if matches or (status FAIL) if no pcurve,
+   nothing done</UserDocu>
       </Documentation>
     </Methode>
     <Methode Name="fixRemoveCurve3d">
       <Documentation>
-        <UserDocu></UserDocu>
+        <UserDocu>Removes 3d curve of the edge if it does not match the vertices
+   Returns: True,  if does not match, removed (status DONE)
+   False, (status OK) if matches or (status FAIL) if no 3d curve,
+   nothing done
+</UserDocu>
       </Documentation>
     </Methode>
     <Methode Name="fixAddPCurve">
       <Documentation>
-        <UserDocu></UserDocu>
+        <UserDocu> Adds pcurve(s) of the edge if missing (by projecting 3d curve)
+   Parameter isSeam indicates if the edge is a seam.
+   The parameter <prec> defines the precision for calculations.
```

Comment:
`<prec>` is not a valid text string because the XML parser thinks it opens a new element and then struggles when it finds the next ending element `</UserDocu>` in line 58